packages/bitcore-lib/README.md

```diff
@@ -46,27 +36,71 @@ gulp browser
 
 This will generate files named `bitcore-lib.js` and `bitcore-lib.min.js`.
 
-You can also use our pre-generated files, provided for each release along with a PGP signature by one of the project's maintainers. To get them, checkout the [releases](https://github.com/bitpay/bitcore/blob/master/packages/bitcore-lib/CHANGELOG.md).
-
-## Development & Tests
```

Comment:
I think this just needs some updating instead of a full removal
```
clone bitcore
npm install
cd packages/bitcore-lib
```
...
run tests with `npm test` (or whatever) instead of gulp (it still uses gulp, but it uses it via the test command and bitcore-build)